src/middleware/auth.middleware.ts

```diff
@@ -1,13 +1,23 @@
 import { NextFunction, Request, RequestHandler, Response } from 'express'
 import { AuthOptions } from 'web-security-node'
 
-import { SEARCH_COMPANY_URI } from 'app/paths'
+import { HEALTHCHECK_URI, ROOT_URI, SEARCH_COMPANY_URI, WHO_TO_TELL_URI } from 'app/paths'
 import UriFactory from 'app/utils/uri.factory'
 
+const USER_AUTH_WHITELISTED_URLS: string[] = [
+  `${ROOT_URI}/`,
```

Comment:
For the root URI (landing page), `req.url` is "/close-a-company/". So thought this was cleaner than messing with any config or changing the consts in `paths.ts`